### PR TITLE
BOM-1351: Added django 2.x workers to travis 

### PR DESCRIPTION
**Pull request by iamsobanjaved**
JIRA: [BOM-1351](https://openedx.atlassian.net/browse/BOM-1351)

- Added django 2.x workers to travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -59,3 +59,51 @@ matrix:
         - pip install --upgrade codecov
         - make exec-coverage
         - codecov
+
```
Comment (Ayub-Khan) [inline .travis.yml]: Need to remove the repetition

File: requirements/pip_tools.txt [hunk 2]
```diff
@@ -5,5 +5,5 @@
 #    make upgrade
 #
 click==7.0                # via pip-tools
-pip-tools==4.4.1
+pip-tools==4.5.1          # via -r requirements/pip_tools.in
```
Comment (Ayub-Khan) [inline requirements/pip_tools.txt]: You might want to run the 'make upgrade' command again for 4.5.1 to remove unnecessary changes.

File: .travis.yml [hunk 3]
```diff
@@ -59,3 +59,51 @@ matrix:
...
+    - env: TOXENV=django20
+
+      # Doing so is a waste of time since they won't be used.
```
Comment (awais786) [inline .travis.yml]: remove this

Comment (iamsobanjaved) [reply .travis.yml]: Install statement?
